cibyl/plugins/openstack/__init__.py

```diff
@@ -68,6 +69,8 @@ def extend_models(self):
                     self.plugin_attributes_to_add)
             setattr(job_class, 'add_deployment',
                     add_deployment)
+        FeaturesLoader.features_locations.append(f"{__path__[0]}/features")
```

Comment:
Another reason as to pass the path on the constructor. How easy it is to miss this change during debugging.

Reply:
I'm not sure about this one, this follows the way we have been using the plugins. To make it more explicit I'll move it into a different method, for better visibility.